src/components/Shortcuts/shortcut.styl

```diff
@@ -0,0 +1,13 @@
+.shortcut-actions-menu
+    top 0.5rem
+    right 1rem
+    opacity 0
+
+.shortcut-actions-menu:hover
+    opacity 1
```

Comment:
is it sure we want this behavior? There is no "hover" in mobile, so how we handle that on this device? The menu is always shown, isn't it? So why not always showing it also on desktop?